nwb_conversion_tools/datainterfaces/behavior/movie/moviedatainterface.py

```diff
@@ -42,6 +47,25 @@ def __init__(self, file_paths: list):
         assert HAVE_OPENCV, INSTALL_MESSAGE
         super().__init__(file_paths=file_paths)
 
+    def get_metadata_schema(self):
+        metadata_schema = super().get_metadata_schema()
+        metadata_schema["properties"]["behavior"] = get_base_schema(tag="behavior")
+        metadata_schema["properties"]["behavior"]["required"] = ["Movies"]
+        metadata_schema["properties"]["behavior"]["properties"] = dict(
+            Movies=dict(type="array", items=get_schema_from_hdmf_class(ImageSeries))
```

Comment:
```suggestion
        metadata_schema["properties"]["Behavior"] = get_base_schema(tag="Behavior")
        metadata_schema["properties"]["Behavior"]["required"] = ["Movies"]
        metadata_schema["properties"]["Behavior"]["properties"] = dict(
            Movies=dict(type="array", items=get_schema_from_hdmf_class(ImageSeries))
```
To be consistent with the Ecephys metadata, starting each key with a capital letter